### PR TITLE
Install python3

### PR DESCRIPTION
**Pull request by susannasiebert**
This will allows us to invoke the add bam readcount helper script with `python3 /usr/bin/add_bam_reacount_to_vcf_helper.py `.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -152,7 +154,7 @@ COPY bam_readcount_helper.py /usr/bin/bam_readcount_helper.py
 COPY add_bam_reacount_to_vcf_helper.py /usr/bin/add_bam_reacount_to_vcf_helper.py
```
Comment (dufeiyu) [inline Dockerfile]: There is a typo **reacount** in python script name.

**Comment by dufeiyu** [comment]
Actually that deployed python script under docker-cle gets wrong name too and need change:
docker-cle/add_bam_reacount_to_vcf_helper.py

**Comment by susannasiebert** [comment]
Yeah, I forgot to commit that part. It's fixed now.

**Review by dufeiyu — APPROVED** [review]
+1

**Pull request merged**